litewallet/UnstoppableDomainView.swift

```diff
@@ -0,0 +1,117 @@
+import Combine
+import SwiftUI
+
+struct UnstoppableDomainView: View {
+	// MARK: - Combine Variables
+
+	@ObservedObject
+	var viewModel: UnstoppableDomainViewModel
+
+	@State
+	private var didReceiveLTCfromUD: Bool = false
+
+	@State
+	private var shouldDisableLookupButton: Bool = true
+
+	@State
+	private var didStartEditing: Bool = false
+
+	@State
+	private var shouldStartScroll: Bool = false
+
+	init(viewModel: UnstoppableDomainViewModel) {
+		self.viewModel = viewModel
+	}
+
+	var body: some View {
+		GeometryReader { _ in
+
+			ZStack {
+				VStack {
+					Spacer()
+					HStack {
+						if viewModel.isDomainResolving {
+							ProgressView()
+								.padding(.all, swiftUICellPadding)
+						} else {
+							VStack {
+								ZStack {
+									AddressFieldView(placeholder: viewModel.placeholderString,
+									                 text: $viewModel.searchString)
+										.onTapGesture {
+											didStartEditing = true
+										}
+										.frame(height: 45.0, alignment: .leading)
+										.padding(.leading, swiftUICellPadding)
+								}
+							}
+						}
+
+						Spacer()
+					}
+					.background(
+						Color.white.clipShape(RoundedRectangle(cornerRadius: 8.0))
+					)
+					.padding([.leading, .trailing], swiftUICellPadding)
```

Comment:
Awesome! But I have questions. What are leading and trailing? Thank you! @kcw-grunt

Reply:
Hey @josikie 
The `.leading and .trailing` are layout settings for padding (in this case)